Scripts/script-Ping.yml

```diff
@@ -16,10 +16,10 @@ script: |-
 
       verbose = demisto.get(demisto.args(),'verbose')
       if verbose and verbose.lower() == 'false':
```

Comment:
@aviadl I also don't think we need to verbose argument.. this is why we have the raw-response... this script is a bit strange.. :)